### ISSUE TITLE
Alternative method of opening bookmarks sidebar

### ISSUE THREAD

**Issue by sam-lunt** [tridactyl/tridactyl#1181]
The FAQ mentions unbinding `<C-b>` if you want to maintain the default Firefox behavior of opening the bookmarks sidebar. I was hoping there was a another way to quickly open the sidebar, but I wasn't able to find one in the help section. An Ex command would be my ideal, but a keyboard shortcut would also work. If the Firefox API does not expose a way to do this, I apologize for the spurious issue.

Also, thanks for all the work you all do, this is a great extension.

**Comment by bovine3dom** [comment]
I don't think there is any way for us to open the bookmarks sidebar. See #760 - we have issues open with Firefox to allow us to trigger actions that could be done from keyboard shortcuts.

In the meantime, have you seen the `bmarks` command?

**Comment by sam-lunt** [comment]
I was afraid it was something like that. Thanks for the response.

I did try the `bmarks` command, but the issue is that I have a decent number of bookmarks organized into folders. Usually when I'm opening something I've bookmarked, it's for one of two reasons: I want to see everything from a specific folder, or I know the general category of something but forget the exact site/page name. As far as I can tell, `bmarks` doesn't display the folders, so it doesn't quite fit that workflow. I appreciate the suggestion though.
